base_technical_user/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    user_tech_id = fields.Many2one(
+        comodel_name="res.users",
+        string="Technical User",
+        help="This user can be used by process for technical purpose",
+        domain="[('company_id', '=', id), ('active', '=', False)]")
```

Comment:
If we don't want to take the risk to disable the user (regarding [this comment](https://github.com/OCA/server-tools/pull/858#discussion_r125591029)), could you remove the domain on active = false?